Source/WebKit/UIProcess/API/gtk/WebKitWebViewBasePrivate.h

```diff
@@ -52,7 +52,7 @@ void webkitWebViewBaseSetTooltipText(WebKitWebViewBase*, const char*);
 void webkitWebViewBaseSetTooltipArea(WebKitWebViewBase*, const WebCore::IntRect&);
 void webkitWebViewBaseSetMouseIsOverScrollbar(WebKitWebViewBase*, WebKit::WebHitTestResultData::IsScrollbar);
 void webkitWebViewBaseForwardNextKeyEvent(WebKitWebViewBase*);
```

Comment:
Would the same strategy suffice to fix key events too?

Reply:
Sure, added (and it's simpler because key events shouldn't get compressed/coalesced)